internal/sentry.go

```diff
@@ -22,3 +23,20 @@ func GetSentryHubFromContextOrDefault(ctx context.Context) *sentry.Hub {
 	}
 	return hub
 }
+
+// ReportPanicsToSentry checks for panics by calling recover, reports any panic found to
+// sentry, and then reraises the panic. To have tracebacks included in the report, make
+// sure you call panic with an error, not a string.
```

Comment:
Errors are basically strings, so this doesn't make sense.

Reply:
Poorly worded, but I have in mind this logic: https://github.com/getsentry/sentry-go/blob/eec094e9470dd3855eaf47b025d853bcbc13df68/client.go#L438-L447